meson_options.txt

```diff
@@ -25,5 +25,5 @@ option('examples',
        type: 'boolean', value: true,
        description: 'Whether to build the examples')
 option('tests',
-       type: 'boolean', value: true,
+       type: 'boolean', value: false,
```

Comment:
Why false by default now?

Reply:
Because the tests require access to some static code, so when compiling with the tests enabled, a new function is exported that just calls that static function. But in normal circumstances we don't want to have that function exported, so I decided to disable the tests by default.

Reply:
Now that I used a private header, this change isn't needed anymore. Reverted.